easybuild/easyconfigs/c/CrossTalkZ/CrossTalkZ-1.0-foss-2016a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'CrossTalkZ'
+version = '1.4'
+
+homepage = 'http://sonnhammer.sbc.su.se/download/software/CrossTalkZ/'
+description = """
+CrossTalkZ is a statistical method and software to assess the significance of crosstalk enrichment between pairs of gene or protein groups in large biological networks.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'optarch': True, 'usempi': True, 'pic': True}
+
+source_urls = [
+    'http://sonnhammer.sbc.su.se/download/software/CrossTalkZ/'
+]
+sources = ['%(name)s_%(version)s.tgz']
+
+dependencies = [('Boost', '1.60.0')]
+
+start_dir = 'src'
+
+configopts = " -DBoost_NO_BOOST_CMAKE=ON -DBoost_NO_SYSTEM_PATHS=ON -DBOOST_ROOT=$EBROOTBOOST"
+
+builddependencies = [('CMake', '3.4.3', '', ('foss','2016a'))]
```

Comment:
no need to specify `foss` here, since that's the parent toolchain, just use `('CMake', '3.4.3')`